aten/src/ATen/cpu/vec/vec256/vec256_int.h

```diff
@@ -745,6 +745,259 @@ class Vectorized<int8_t> : public Vectorizedi {
   Vectorized<int8_t> le(const Vectorized<int8_t>& other) const;
 };
 
+template <>
+class Vectorized<uint8_t> : public Vectorizedi {
```

Comment:
@alexsamardzic, as @jgong5 [mentioned here](https://github.com/pytorch/pytorch/pull/89284#discussion_r1028980166), can you please add a common class for both `int8` & `uint8`, and then add derived classes for `int8` & `uint8_t`? Thanks!

Reply:
I think this PR may be fine as-is. 
I'd try to merge this one now with minimal changes, and then we can file an issue describing these further improvements.

Reply:
Sure, thanks! :)